src/storage/Manager.h

```diff
@@ -99,7 +113,7 @@ class Manager final {
     friend class storage::detail::ExpirationTimer;
     void RunExpireThread();
     void StartExpirationTimer();
-    std::jthread expiration_thread;
+    zeek::jthread expiration_thread;
```

Comment:
Minor: We don't use explicit `zeek::` namespace elsewhere, but I'm actually never sure what the best pattern is to use. For that one, it probably makes sense to keep it explicit?